### PR TITLE
feat(plugin): add KyzaMode

### PR DESCRIPTION
**Pull request by Ruthenic**

### CODE REVIEW

**Comment by ArjixWasTaken** [comment]
:thumbsup:

**Comment by ArjixWasTaken** [comment]
merge wen

**Comment by Ruthenic** [comment]
my thoughts exactly

File: src/plugins/kyzaMode.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    " i!": " I!",
+    " i.": " I.",
+    "i'll": "I'll",
+    "i'd": "I'd",
+    "Https://": "https://",
+}; //dictionary of words and replacements
+
+export default definePlugin({
+    name: "KyzaMode",
+    description: "Fixes your goofy grammar",
```
Comment (adryd325) [inline src/plugins/kyzaMode.ts:16]: make sure to add
```dependencies: ["MessageEventsAPI"],```

**Review by adryd325 — CHANGES_REQUESTED** [review]

**Comment by Ruthenic** [comment]
if the fucking uwuifier can get merged this can get merged 
image

**Comment by ArjixWasTaken** [comment]
you didn't even follow the suggestion by adryd from 4 days ago

**Comment by Ruthenic** [comment]
i did i just didn't push the commit until now lol

**Comment by Ruthenic** [comment]
(and it's broken anyways because i forgot a comma :skull:)

**Comment by somerandomcloud** [comment]
Im just wondering why the name

**Comment by ArjixWasTaken** [comment]
> Im just wondering why the name

I assume the name stuck from the BD plugin
Which might be named after a person that was a grammar Nazi

(I am making assumptions, nothing I said is confirmed)

**Comment by emir4169** [comment]
can i put this in a dumb github repo for 3rd party plugins

File: src/plugins/kyzaMode.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    authors: [{
+        name: "DustyAngel47",
+        id: 714583473804935238n
+    }],
+    dependencies: ["MessageEventsAPI"],
+
+    getPunctuation(msg) {
```
Comment (luximus-hunter) [inline src/plugins/kyzaMode.ts:23]: Maybe rename to addPunctuationm, as it doesn't return the current punctuation>

Comment (ArjixWasTaken) [reply src/plugins/kyzaMode.ts:23]: It is not named `getCurrentPunctuation`, it is named `getPunctuation`, and it does just that.
It also does not "add", it only "gets", "add" that would mean it returns the message + punctuation, but what it does is just return the punctuation.

**Pull request closed without merging**